microceph/client/client.go

```diff
@@ -50,6 +54,40 @@ func GetConfig(ctx context.Context, c *client.Client, data *types.Config) (types
 	return configs, nil
 }
 
+// IsForwardedRequest determines if this request has been forwarded from another cluster member.
+func IsForwardedRequest(r *http.Request) bool {
+	return r.Header.Get("User-Agent") == request.UserAgentNotifier
+}
+
+func ForwardConfigRequestToClusterMembers(s *state.State, r *http.Request, data *types.Config, handle func(ctx context.Context, c *client.Client, data *types.Config) error) (error) {
```

Comment:
I don't think we want to call this a config forwarding thing; it's more of a generic "please restart this kind of deamon" thing

Reply:
Have abstracted the restart the daemons to a separate API.